.pre-commit-config.yaml

```
@@ -26,21 +21,28 @@ repos:
 #        -   --unsafe-fixes
     -   id: ruff-format
 -   repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v1.6.1
+    rev: v1.11.0
     hooks:
     -   id: mypy
         additional_dependencies: [
-            numpy==1.26.*,  # pre-commit CI does not provide numpy 1.23.*
+            numpy==1.23.*,
```

now using 1.26 (also pre-commit CI does not support numpy 1.23)
```suggestion
            numpy==1.26.*,
```